### PR TITLE
Lesson 1 4

### PR DESCRIPTION
**Pull request by Anakondos**
Lifting State
Controlled Components
Props handling

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -4,16 +4,24 @@ import AddTodoForm from './AddTodoForm';
 
 
 function App() {
+
  
     const [newTodo, setNewTodo] = React.useState('');
  
-
+    /*1.4 new state variable named todoList
```
Comment (Atos20) [inline src/App.js]: I would delete this comment as this one seems to be only the instructions for the assignment.

File: src/App.js [hunk 2]
```diff
@@ -4,16 +4,24 @@ import AddTodoForm from './AddTodoForm';
...
+
  
     const [newTodo, setNewTodo] = React.useState('');
  
-
+    /*1.4 new state variable named todoList 
+    with setter setTodoList and default value of an empty Array*/
+    const [todoList, setTodoList] = React.useState([]);
+    
+    // const handleSearch = (event) => {
```
Comment (Atos20) [inline src/App.js]: commented code should be deleted unless it adds value to the code.

File: src/TodoList.js [hunk 3]
```diff
@@ -2,25 +2,26 @@ import React from 'react';
...
-    {
-      id: 2, 
-      title: "submit assigment"
-    },
-    {
-      id: 3,
-      title: "Start assignment"
-    }
-  ];
+// const todoList = [
```
Comment (Atos20) [inline src/TodoList.js]: let's delete these commented code

File: src/AddTodoForm.js [hunk 4]
```diff
@@ -1,14 +1,24 @@
...
+    }
+    
     
     const handleAddTodo = (event) => {
         event.preventDefault();
-        const todoTitle = event.target.title.value;
-        props.onAddTodo(todoTitle);
-        console.log('This is  todoTitle = ', todoTitle);
-        
+        // const todoTitle = event.target.title.value;
```
Comment (Atos20) [inline src/AddTodoForm.js]: delete line 18

File: src/TodoList.js [hunk 5]
```diff
@@ -16,13 +16,18 @@ const todoList = [
       title: "Start assignment"
     }
   ];
-const TodoList = (props) => {
+
```
Comment (Atos20) [inline src/TodoList.js:5]: I belive the assignment requires you to delete this `todoList` variable on line 5

File: src/AddTodoForm.js [hunk 6]
```diff
@@ -1,23 +1,42 @@
...
+        onAddTodo({
+            title: todoTitle,
+            id: Date.now(),
+        });
+
+        // console.log('This is  todoTitle = ', todoTitle);
         
-        event.target.reset();
+        // event.target.reset();
```
Comment (Atos20) [inline src/AddTodoForm.js]: line 26 also needs to go

**Review by Atos20 — CHANGES_REQUESTED** [review]
good Job Anton, you just need to clean up your code before merging your branch. Please ping me on slack once you have made the required changes so I can submit your assignment review.

File: src/App.js (hunk 1 above)
Comment (Anakondos) [reply src/App.js]: done

File: src/TodoList.js (hunk 5 above)
Comment (Anakondos) [reply src/TodoList.js:5]: correct, thank you

**Review by Atos20 — APPROVED** [review]
Great work!

**Pull request merged**